commands/remove.rb

```diff
@@ -34,17 +45,42 @@ def self.remove(pkg, verbose)
     # Remove the files and directories installed by the package.
     unless pkg.is_fake?
       Dir.chdir CREW_CONFIG_PATH do
-        # Remove all files installed by the package.
+        # Remove all files installed by the package in CREW_PREFIX and
+        # HOME.
```

Comment:
Won't this conflict with our current config directory code which allows the user to choose if they want to remove it or not?

Reply:
Hmmm... what do you think @uberhacker?

Just grepping for `/home/chronos/user` in the manifests dir it does look like there are a decent number of packages that put files in `#{HOME}`.

Reply:
Yes, we have `HOME` stuff in the manifest filelists so we need to remove that too during package removal.